jsk_pcl_ros/src/attention_clipper_nodelet.cpp

```diff
@@ -390,21 +389,19 @@ namespace jsk_pcl_ros
         // check transform cache
         if (transforms.find(frame_id) == transforms.end()) {
           tf::StampedTransform new_transform = lookupTransformWithDuration(
-            tf_listener_,
-            frame_id, msg->header.frame_id,
-            msg->header.stamp,
-            ros::Duration(1.0));
-          transforms[frame_id] = new_transform;
+            /*listener=*/tf_listener_,
+            /*to_frame=*/frame_id,                // box origin
+            /*from_frame=*/msg->header.frame_id,  // sensor origin
+            /*time=*/msg->header.stamp,
+            /*duration=*/ros::Duration(1.0));
+          transforms[frame_id] = new_transform; // sensor to box
```

Comment:
It should be box to sensor isn't it?
I'm not sure what is `to_frame` and `from_frame`...